designs/developer/add-45/sections/Adv-Health_Checks.md

```diff
@@ -5,4 +5,63 @@ version: 4.5
 
 # Add Flows - Advanced Options - Health Checks
 
-coming soon
+Health Checks is a link within the Advanced Options section for the Import from Git and Deploy Image +Add flows. The Health Checks link goes after the Routing link and before the Build Configuration link in the list of Advanced Option links.
+
+![Health Checks link](img/Select_Health_Checks.png)
+
+
+When a user clicks on the Health Checks link, the Health Checks section is expanded within the Advanced Options section.
+
+![Health Checks link clicked](img/Select_Health_Checks_2.png)
+
+When the Health Checks section is expanded there are 3 probes listed (Readiness, Liveness, and Startup) with a Add <probe name> link associated with each probe.
+
+![Health Checks options](img/Select_Add_Probe.png)
+
+When an  Add <probe name>  link is clicked, the probe form expands exposing all of the fields associated with the probe.  The user can confirm or cancel the adding of the Health Check using the check or x.  There are 3 variations on the Health Checks form which are based on the Type that is selected. The default probe type is HTTP GET. See the Health Checks In-Line form section below for these variations.
+
+![Health Checks inline form](img/Add_Probe_Inline_Form_Expands.png)
```

Comment:
I wonder if using the inline Create and Cancel buttons over the X and checkmark would maintain better consistency with the OpenShift design conventions: http://openshift.github.io/openshift-origin-design/conventions/documentation/edit.html

Reply:
@lizsurette We discussed this in depth. There was feedback similar to your suggestion. Sunil even did some alternative mocks. In the end, we chose this design to differentiate from the create/cancel actions since the check/x just adds and or clears the inline form. Our plan is definitely to track this for usability. Another interesting idea was to use icons to represent add and clear.

Reply:
oh I didn't realize that OpenShift already had something called an inline form!  Agreed @beaumorley, this `form-within-a-form` has been utilized for a couple of releases now.  Agreed we can continue to get feedback on it, but as you mentioned we just attempted some visual design options after getting some feedback from chris & brie?,, but ended up coming back to where we started.

I'm fine with `form-within-a-form`

Reply:
Makes sense to me too!! Thanks for following up on this @beaumorley and @serenamarie125 !